### PR TITLE
fixes: duplicate entries

### LINKED ISSUE #7: bug: the subcommand `add` creates duplicate entries in TUI.
When the subcommand `add` is run multiple times on the same directory(same absolute path).
It creates multiple entries of it in TUI.

pms add subcmd

### PR DESCRIPTION
**Pull request by sidkhuntia**
fixes #7

### CODE REVIEW

File: utils/config.go [hunk 1]
```diff
@@ -126,12 +126,17 @@ func AddProject(workingPath string) error {
...
+			return nil
+		}
+	}
+
 	c.Projects = append(c.Projects, newProject)
 	byteData, err := json.MarshalIndent(c, "", "    ")
 	if err != nil {
 		return err
 	}
-
```
Comment (jammutkarsh) [inline utils/config.go]: Please don't delete this line.

**Comment by jammutkarsh** [comment]
1. Always add your signature when committing: `git commit -s -m "task: title" -m "elaborate description." Why? because most oss project requires contributors this.

2. Merge atomic commits. Why? In the long run, small commits like `fixed this`, `patched that` doesn't add much value, when we want to roll back.
What would be useful is know what as a whole and when. So when something breaks, we simply roll back to a single commit, instead of dangling and checking across multiple commits.

Make atomic commits for your personal rollbacks and push these changes in a PR for atomic review(helpful sometimes). Finally merge them all together when you have reached the end stage.

**Comment by jammutkarsh** [comment]
One more thing I noticed now. It is not advised to make changes to the main branch. In your fork, you made changes to `sidkhuntia:main` then with the same branch.

Refer
1. https://opensource.guide/how-to-contribute/#opening-a-pull-request
2. [FreeCodeCamp](https://www.freecodecamp.org/news/how-to-contribute-to-open-source-projects-beginners-guide/#:~:text=How%20to%20Submit%20a%20Pull%20Request)

This is a guideline for collaborative work. [Read more](https://softwareengineering.stackexchange.com/questions/441647/when-should-i-create-another-branch-in-git-and-when-should-i-use-the-main-branch).

Please create another branch and make a PR.

**Comment by sidkhuntia** [comment]
Thanks for help, I will be closing this PR and creating a new PR

**Pull request closed without merging**